SPECS/expat/expat.spec

```diff
@@ -55,17 +53,22 @@ rm -rf %{buildroot}/*
 %license COPYING
```

Comment:
Please move the license file to the `libs` subpackage.  In a hypothetical situation where a user has only `expat-libs` installed, we would not be meeting the redistribution clause of the MIT license. Since the `libs` subpackage is installed as a transitive dependency of all subpackages in this spec, it's fine to just put the license in that subpackage.